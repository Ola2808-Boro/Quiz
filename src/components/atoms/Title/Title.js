import styled from "styled-components";
import { device } from "helpers/breakpoints";
export const StyledTitle = styled.h1`
  font-size: ${({ theme }) => theme.fontSize.l};
  color: ${({ theme }) => theme.colors.text};
  letter-spacing: 2px;
  padding: 1rem;

  @media ${device.mobileS} {
    font-size: ${({ theme }) => theme.fontSize.xxxl};
  }

  @media ${device.mobileL} {
    font-size: ${({ theme }) => theme.fontSize.xxxl};
  }
  @media ${device.tablet} {
    font-size: ${({ theme }) => theme.fontSize.lgrand};
  }
  @media ${device.laptop} {
    font-size: ${({ theme }) => theme.fontSize.xlgrand};
  }
  @media ${device.desktop} {
    font-size: ${({ theme }) => theme.fontSize.huge};
  }
`;
