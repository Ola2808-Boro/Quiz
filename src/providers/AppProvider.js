import React from 'react';
import { GlobalStyle } from 'assets/styles/GlobalsStyle'
import { ThemeProvider } from 'styled-components';
import 'assets/styles/fonts.css'
import {theme} from 'assets/styles/Theme'


const AppProviders = ({ children }) => {
    return (
 
          <ThemeProvider theme={theme}>      
                <GlobalStyle />
                {children}
          </ThemeProvider>
 
    );
  };
  
  export default AppProviders;