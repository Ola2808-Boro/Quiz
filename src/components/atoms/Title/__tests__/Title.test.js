import React from "react";
import "jest-styled-components";
import renderer from "react-test-renderer";
import AppProvider from "providers/AppProvider";
import { StyledTitle } from "components/atoms/Title/Title";
describe("Test style", () => {
  it("default style", () => {
    const tree = renderer
      .create(
        <AppProvider>
          <StyledTitle />
        </AppProvider>
      )
      .toJSON();
    expect(tree).toMatchSnapshot();
  });
});
