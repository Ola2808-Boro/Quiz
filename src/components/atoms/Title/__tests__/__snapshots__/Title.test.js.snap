// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Test style default style 1`] = `
<h1
  className="kERUH"
/>
`;
